waddrmgr/manager_test.go

```diff
@@ -1222,12 +1248,12 @@ func testNewAccount(tc *testContext) bool {
 	return true
 }
 
-// testLookupAccount tests the basic account lookup func of the address manager works
-// as expected.
+// testLookupAccount tests the basic account lookup func of the address manager
+// works as expected.
 func testLookupAccount(tc *testContext) bool {
 	// Lookup accounts created earlier in testNewAccount
 	expectedAccounts := map[string]uint32{
-		waddrmgr.DefaultAccountName:      waddrmgr.DefaultAccountNum,
+		"":                               waddrmgr.DefaultAccountNum,
```

Comment:
Yeah, the test case was being affected by the `testRenameAccount`. Fixing it to test both cases independently.